DOC.md

```diff
@@ -378,6 +379,81 @@ s("trig", {
 If the function only performs simple operations on text, consider using
 the `lambda` from [`luasnip.extras`](#extras)
 
+# POSTFIX SNIPPET
+
+Postfix snippets, famously used in 
+[rust analyzer](https://rust-analyzer.github.io/) and various IDEs, are a type
+of snippet, which alters text before the snippets trigger. While these
+can be implemented using regTrig snippets, this helper makes the process easier
+in most cases.
+
+The simplest example, which surrounds the text preceeding the `.br` with
+brackets `[]`, looks like:
+
+```lua
+
+      postfix(".br", {
+              f(function(_, parent)
+                    return "[" .. parent.postfix_match .. "]"
```

Comment:
Would you update this to env (ofc) and use `parent.snippet` instead of `parent`? (in this case it's the same table, but there are snippets where they are not, and I'd rather the examples still work then).
Also, could you add one example with lambda? (is it obvious I like them yet? :P)

Reply:
Oh crap I missed one! 
And absolutely about the lambda lol :)

Reply:
Alright, Those examples are in!

Reply:
Nice, that looks good :+1: